stdlib/list.mli

```diff
@@ -101,6 +101,10 @@ val concat : 'a list list -> 'a list
 val flatten : 'a list list -> 'a list
 (** An alias for [concat]. *)
 
+val iota : ?start:int -> ?step:int -> int -> int list
+(** [List.iota ?start ?step len] returns a list containing the elements
+    [[start; start+step; ...; start+(len-1)*step]], where [start] and [step]
+     default to [0] and [1], respectively. *)
```

Comment:
You forgot to document the `invalid_arg` case when `start` is negative.

Reply:
You're right, thanks. As it stands, iota won't used anyway, so no need for a new commit right now - or would you like to see one?